RELEASE.md

```diff
@@ -1,5 +1,13 @@
 # Release notes
 
+## 1.2.4
+
+* Updates both WPF and UWP with a new design that adapts to work well for any screen size.
+    * Adds a `ModernMapPanel` custom layout panel to facilitate consistent, responsive design for UWP and WPF.
+    * Refactors styles and related XAML.
+    * Switches to vector icons where possible.
```

Comment:
My gut says that because this release is pushing out such a significant overhaul to the UI, it's worth calling this version **1.3**, but I know that diverges from the version numbering we had planned. @esreli @nCastle1 what do you guys think?